lib/wallet/nodeclient.js

```diff
@@ -211,6 +211,20 @@ class NodeClient extends AsyncEmitter {
       return this.emitAsync('block rescan', entry, txs);
     });
   }
+
+  /**
+  * Get relevant node info for wallet
+  * @returns {Object}
+  */
+
+  getInfo() {
+    return {
```

Comment:
The result would be different here between `client.js` and `nodeclient.js`. For example, `client.js` would give back _all_ of the info, and `nodeclient.js` gives back only this portion. The code in `http.js` for `get('/'` could be moved to `node.getInfo` and both could call it, and thus the results would be the same. Alternatively, the client could have `isPruned` and `isSPV` methods, and likewise the results would be the same for both `nodeclient.js` and `client.js`.